eslint.config.ts

```diff
@@ -15,8 +13,9 @@ const __filename = fileURLToPath(import.meta.url);
 const __dirname = dirname(__filename);
 const gitignorePath = resolve(__dirname, '.gitignore');
 
-export default tseslint.config(
+export const config: ReturnType<typeof tseslint.config> = tseslint.config(
```

Comment:
AFAICT the export isn't needed here.

```suggestion
const config: ReturnType<typeof tseslint.config> = tseslint.config(
```

Reply:
Oh thanks you are right 👍